docs/guides/cli-completions.md

```diff
@@ -0,0 +1,39 @@
+# CLI Completions
+`lockbook` ships with a `_lockbook` tab completions file.
```

Comment:
`_lockbook` & `FPATH` are zsh specific ideas. Maybe what you want to express here is that we ship with a thin completion file for zsh, bash, & fish which calls out to our cli to perform static and dynamic completions. And that this is powered by the cli-rs crate we wrote. You can consider linking to the crate as well as our blog post.